components/configSwizzleManager.ts

```diff
@@ -280,7 +280,7 @@ export function getVersionedConfig<T = unknown>(
     if (
         // is this scpc, do we have a scpc config?
         gameVersion === "scpc" &&
-        Object.prototype.hasOwnProperty.call(configs, `Scpc${config}`)
+        Object.hasOwn(configs, `Scpc${config}`)
```

Comment:
Do we really need to do a `hasOwn` here? Does it matter if it is inherited, undefined check suffices? I feel this is a true for a lot of locations where we use hasOwn.